rqt_bag/src/rqt_bag/plugins/message_view.py

```diff
@@ -47,19 +49,20 @@ def __init__(self, timeline, topic):
         self.timeline = timeline
         self.topic = topic
 
-    def message_viewed(self, bag, msg_details):
+    def message_viewed(self, bag, entry, ros_message, msg_type_name, topic):
```

Comment:
minor (take it or leave it): Consider changing to keyword arguments
```suggestion
    def message_viewed(self, *, bag, entry, ros_message, msg_type_name, topic):
```

Then child classes calling this function can be simplified if they don't care about some of the arguments, e.g. the `RawView`:

```python
    def message_viewed(self, *, ros_message, msg_type_name, **kwargs):
        super(RawView, self).message_viewed(ros_message=ros_message, msg_type_name=msg_type_name, **kwargs)
```

It could be nice when making changes in the future to this argument list.

On the other hand, switching to keyword args might make things a little more verbose.

Reply:
Sure, I like that. Makes the child classes a bit clearer. Updated.